src/main/scala/uk/gov/hmrc/hmrcfrontend/config/AccountMenuConfig.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.config
+
+import play.api.Configuration
+import javax.inject.Inject
+
+class AccountMenuConfig @Inject() (configuration: Configuration) {
+
+  private val platformHost: Option[String] =
+    configuration.getOptional[String]("platform.frontend.host")
+
+  private val ptaAccountMenuHost: Option[String] =
+    configuration.getOptional[String]("pta-account-menu.host")
+
+  private def baseUrl: String =
+    platformHost
+      .orElse(ptaAccountMenuHost)
+      .getOrElse("")
```

Comment:
@JoPintoPaul / @kyle-bowden I don't know if this is going against the convention of the platform, but should we move the default value into the config instead? And maybe have separate hosts for the different services rather than a shared one, otherwise if you are running the services locally on different ports then some / all won't work

```
pta-account-menu.pertax-frontend.host = ${platform.frontend.host:-localhost:0000}
```

default value there is made up but just to show what I mean, I'm not sure if pertax-frontend is the right service name either here

Reply:
Thank you for the suggestion, the code has now been updated to reflect the change.